### PR TITLE
P1645R1 constexpr for <numeric> algorithms

### LINKED ISSUE #3425: [2019-11 LWG Motion 23] P1645R1 constexpr for <numeric> algorithms
http://wiki.edg.com/pub/Wg21belfast/StrawPolls/P1645R1__constexpr_for__numeric__algorithms.html

 This resolves NB comment US320

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #3425.

Fixes cplusplus/nbballot#316
Fixes cplusplus/papers#432

### CODE REVIEW

File: source/support.tex [hunk 1]
```diff
@@ -580,6 +580,7 @@
   // also in any C++ library header from \tref{headers.cpp} or any C++ header for C library facilities from \tref{headers.cpp.c}
 #define @\defnlibxname{cpp_lib_constexpr_dynamic_alloc}@           201907L // also in \libheader{memory}
 #define @\defnlibxname{cpp_lib_constexpr_invoke}@                  201907L // also in \libheader{functional}
+#define @\defnlibxname{cpp_lib_constexpr_numeric}@                 201911L // also in \libheader{numeric}
```
Comment (jensmaurer) [inline source/support.tex]: The paper says "cpp_lib_constexpr_numeric_algorithms".

**Comment by jensmaurer** [comment]
... and please revert the order of the two lines in the commit comment: Paper number and title first; NB comment references in the body of the commit message.

File: source/support.tex (hunk 1 above)
Comment (CaseyCarter) [reply source/support.tex]: Both [the revision of `d1645r1.html` we approved in LWG](http://wiki.edg.com/pub/Wg21belfast/LibraryWorkingGroup/d1645r1.html) and the [P1645R1 attached to the StrawPolls page](http://wiki.edg.com/pub/Wg21belfast/StrawPolls/P1645R1__constexpr_for__numeric__algorithms.html) use `__cpp_lib_constexpr_numeric`.

Comment (jensmaurer) [reply source/support.tex]: @CaseyCarter , you're right. I have no idea where I looked to find the other spelling.

**Comment by burblebee** [comment]
> ... and please revert the order of the two lines in the commit comment: Paper number and title first; NB comment references in the body of the commit message.

I did my best to follow the guiodelines at https://github.com/cplusplus/draft/wiki/Commit-message-format, which suggest NB comments should be presented with "https://github.com/cplusplus/draft/wiki/Commit-message-format".  If that is not correct, please fix the documentation and I will happily follow suit.  I.e. I would like to know the correct representation before guessing again at what it should be.  Some examples would help!

**Comment by jensmaurer** [comment]
@burblebee, please read the text on quoted wiki page carefully:

> For a commit that incorporates a N paper, P paper, LWG issue resolution, CWG issue resolution, or editorial NB comment resolution, commit messages for these take the form:

This is not an editorial NB comment resolution, this is a P paper being applied (that also happens to fix one or more NB comments).
So, please use "P1234R5 Title From Paper" and keep reading for the "Also fixes".

**Comment by burblebee** [comment]
@jensmaurer 
> @burblebee, please read the text on the quoted wiki page carefully:
> 
> > For a commit that incorporates a N paper, P paper, LWG issue resolution, CWG issue resolution, or editorial NB comment resolution, commit messages for these take the form: ...

Ah, I missed that.  Thank you for pointing this out.  That said, I would have to force push the branch to fix this. @zygoloid, do you want me to fix and force push or do you want to make these changes when you merge?  Note that all of the NB papers I edited will need to be fixed.

**Pull request merged**